en/services/payload_control.md

```diff
@@ -0,0 +1,62 @@
+<a id="MAV_CMD_DO_SET_CAM_TRIGG_INTERVAL"></a>[MAV_CMD_DO_SET_CAM_TRIGG_INTERVAL](../messages/common.md#MAV_CMD_DO_SET_CAM_TRIGG_INTERVAL) | Command to set time interval between camera captures
+<a id="MAV_CMD_DO_SET_CAM_TRIGG_DIST"></a>[MAV_CMD_DO_SET_CAM_TRIGG_DIST](../messages/common.md#MAV_CMD_DO_SET_CAM_TRIGG_DIST) | Command to set distance travelled between camera captures.
+
+
+## Arbitrary/Unknown Payload Commands
+
+The [MAV_CMD_DO_SET_ACTUATOR](#MAV_CMD_DO_SET_ACTUATOR) and [MAV_CMD_DO_SET_SERVO](#MAV_CMD_DO_SET_SERVO) commands allow setting the value of flight controller outputs, without having to know what that specific hardware is. 
+
+> **Tip** Use the [commands for known payload types](#payload-specific-commands) where possible.
+  These are more intuitive for users, and in logs.
+
+`MAV_CMD_DO_SET_ACTUATOR` is better designed and more recent than `MAV_CMD_DO_SET_SERVO`, but has not yet been widely adopted.
+- `DO_SET_ACTUATOR` can in theory be used on any bus - MAIN, AUX, UAVCAN, etc., while `DO_SET_SERVO` can only be used on PWM busses, and is generally hardcoded to map instance numbers to AUX outputs (this is not mandated by the specification, so cannot be relied on).
+- `DO_SET_ACTUATOR` can set multiple outputs in a single message. 
+
+Message | Description
+-- | --
+<a id="MAV_CMD_DO_SET_ACTUATOR"></a>[MAV_CMD_DO_SET_ACTUATOR](../messages/common.md#MAV_CMD_DO_SET_ACTUATOR) | Sets actuators (e.g. servos) to a desired value. The actuator numbers are mapped to specific outputs (e.g. on any MAIN or AUX PWM or UAVCAN) using a flight-stack specific mechanism (for example, parameters).
+<a id="MAV_CMD_DO_SET_SERVO"></a>[MAV_CMD_DO_SET_SERVO](../messages/common.md#MAV_CMD_DO_SET_SERVO) | Sets a servo, identified by a specified instance number, to a specified PWM value.
+
+
+> **Note** **Implementations:**
+  - `MAV_CMD_DO_SET_SERVO` is impemented on both ArduPilot and PX4.
+    In both cases instance numbers map to AUX outputs.
+  - `MAV_CMD_DO_SET_ACTUATOR` is impemented only on PX4 (June 2021).
+    Only three outputs can be set (param 1, 2, 3), and these are mapped by parameters to AUX outputs chosen by the user (though there is nothing to stop them being mapped to other busses).
+	For more information see the [PX4 User Guide](https://docs.px4.io/master/en/payloads/#mission-triggering).
+
+## Payload Deployment Commands
+
+These commands can be used to deploy a payload at a specfic location, controlling the approach and land behaviour.
+
+Message | Description
+-- | --
+<a id="MAV_CMD_PAYLOAD_CONTROL_DEPLOY"></a>[MAV_CMD_PAYLOAD_CONTROL_DEPLOY](../messages/common.md#MAV_CMD_PAYLOAD_PREPARE_DEPLOY) | Control payload deployment (abort, continue, switch to palload deployment mode, delete first payload deployment request, delete all payload deployment requests.
```

Comment:
@LorenzMeier These were added in your PR back in https://github.com/mavlink/mavlink/pull/235 
Do you know if they were ever implemented, and if so, by who?

Reply:
Note, MAV_CMD_PAYLOAD_CONTROL_DEPLOY deprecated - https://github.com/mavlink/mavlink/pull/1648